app/google-services.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Remove from public repo see this link:https://stackoverflow.com/questions/37358340/should-i-add-the-google-services-json-from-firebase-to-my-repository